internal/nginx/manager.go

```diff
@@ -289,6 +289,7 @@ func (lm *LocalManager) Reload() error {
 	}
 
 	lm.metricsCollector.IncNginxReloadCount()
+	lm.metricsCollector.UpdateWorkerProcessCount("new")
```

Comment:
Sorry, I told you it was `new` but it should be `curent` 🙏 
```suggestion
	lm.metricsCollector.UpdateWorkerProcessCount("current")
```